rootfs/etc/profile.d/prompt.sh

```diff
@@ -168,9 +168,13 @@ function geodesic_prompt() {
 		KUBE_PS1_KUBECONFIG_CACHE=""
 	fi
 
+	local kube_prompt=""
+	if typeset -f "${kube_ps1}" > /dev/null; then
```

Comment:
We should be checking `typeset -f` against the function name, no need for interpolation. Also consider using `declare` instead of `typeset`.

```suggestion
	if typeset -f "kube_ps1" > /dev/null; then
```